moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -496,242 +496,244 @@ const ProjectTimeline = ({
-                          size="large"
-                          startIcon={<AddCircleIcon />}
-                          ref={addActionRefPhases}
-                          onClick={props.action.onClick}
-                        >
-                          Add phase
-                        </Button>
-                      );
-                    }
-                  },
-                }}
-                editable={{
-                  onRowAdd: (newData) => {
-                    const newPhaseObject = Object.assign(
-                      {
-                        project_id: projectId,
-                        completion_percentage: 0,
-                        completed: false,
-                      },
-                      newData
-                    );
-
-                    // if necessary, updates existing phases in table to ensure only one is marked "current"
-                    updateExistingPhases(newPhaseObject);
-
-                    const projectUpdateInput = getProjectStatusUpdateObject(
-                      newPhaseObject?.phase_id
-                    );
-
-                    // Execute insert mutation, returns promise
-                    return addProjectPhase({
-                      variables: {
-                        objects: [newPhaseObject],
-                      },
-                    })
-                      .then(() =>
-                        !!newPhaseObject?.is_current_phase
-                          ? updateProjectStatus({
-                              variables: {
-                                projectId: projectId,
-                                projectUpdateInput: projectUpdateInput,
-                              },
-                            })
-                          : true
-                      )
-                      .then(() => {
-                        // Refetch data
-                        refetch();
-                        refetchSummary();
-                      });
-                  },
-                  onRowUpdate: (newData, oldData) => {
-                    const updatedPhaseObject = {
-                      ...oldData,
-                    };
-                    // Array of differences between new and old data
-                    let differences = Object.keys(oldData).filter(
-                      (key) => oldData[key] !== newData[key]
-                    );
-
-                    // Loop through the differences and assign newData values.
-                    // If one of the Date fields is blanked out, coerce empty
-                    // string to null.
-                    differences.forEach((diff) => {
-                      let shouldCoerceEmptyStringToNull =
-                        newData[diff] === "" &&
-                        (diff === "phase_start" || diff === "phase_end");
-
-                      if (shouldCoerceEmptyStringToNull) {
-                        updatedPhaseObject[diff] = null;
+          {table !== "milestones" && (
```

Comment:
github is struggling a bit with the diff, but as far as I can tell the only changes (besides indentation) from here down is the conditional rendering based on table name, right?